libs/vkd3d/shaders/cs_swapchain_fullscreen.comp

```diff
@@ -0,0 +1,21 @@
+#version 450
+layout(local_size_x = 8, local_size_y = 8) in;
+
+layout(set = 1, binding = 0) writeonly uniform image2D FragColor;
+layout(set = 0, binding = 0) uniform sampler2D Tex;
+
+layout(push_constant, std430) uniform Registers
+{
+  uvec2 output_resolution;
```

Comment:
Could we query this with `imageSize` or is there a scenario where the image size and output resolution would differ?

Reply:
Scaling as mentioned earlier.

Reply:
Removed from PR.